docs/user-guides/Connect-a-Wallet.md

```diff
@@ -0,0 +1,30 @@
+# Connect a Wallet
+
+## What You'll Need
+
+A compatible CSPR wallet: either [CasperLabs's Signer](https://www.notion.so/Signer-Guide-eb8c504d067344109351bc0f21ac2412) or [Ledger](https://www.ledger.com/). More wallets are planned for the future.
```

Comment:
I would remove the Ledger for now, as it is not available yet. Maybe "More wallets, including Ledger, are planned for the future" ?

Reply:
Done.